box-share-sdk/src/main/java/com/box/androidsdk/share/fragments/CollaborationsFragment.java

```diff
@@ -40,6 +40,7 @@
     protected TextView mNoCollaboratorsText;
     protected CollaboratorsAdapter mCollaboratorsAdapter;
     protected BoxIteratorCollaborations mCollaborations;
+    protected ArrayList<BoxCollaboration.Role> rolesArr = null;
```

Comment:
Not sure it's good to keep this a separate object since the original item  could be changed. As such it will be better to get it from the object at all times.